app/content/team/index.js

```diff
@@ -71,174 +140,332 @@ function Index() {
+                            style={styles.profilePicture}
+                          />
+                        )}
+                        <View style={styles.penIconContainer}>
+                          <MaterialIcons name="edit" size={24} color="black" />
+                        </View>
+                      </View>
+                    </TouchableOpacity>
+
+                    {/* Name Update input field */}
+                    <BottomSheetTextInput
+                      style={styles.input}
+                      value={newName}
+                      onChangeText={(text) => setNewName(text)}
+                      placeholder="Update team name"
                     />
-                  </Menu>
-                ) : (
-                  <></>
-                )
-              }
-            />
-            <KeyboardAwareScrollView
-              // allows opening links from search results without closing keyboard first
-              keyboardShouldPersistTaps="handled"
-              showsVerticalScrollIndicator={false}
-              stickyHeaderIndices={[3]}
-              refreshControl={
-                <RefreshInvalidate invalidateKeys={invalidateKeys} />
-              }
-            >
-              <View style={{ alignItems: "center" }}>
-                <Image
-                  uri="https://upload.wikimedia.org/wikipedia/en/thumb/1/1b/Oregon_State_Beavers_logo.svg/1200px-Oregon_State_Beavers_logo.svg.png"
-                  style={{ marginTop: 0, width: 131, height: 75 }}
-                />
-              </View>
-              <View style={{ alignItems: "center" }}>
-                <View
-                  style={{
-                    flexDirection: "row",
-                    alignItems: "baseline",
-                  }}
+
+                    {/* Save Button */}
+                    <TouchableOpacity
+                      style={styles.saveChangesButton}
+                      onPress={handleUpdate}
+                    >
+                      <Text style={styles.saveChangesButtonText}>Update</Text>
+                    </TouchableOpacity>
+                  </BottomSheetScrollView>
+                </BottomSheetWrapper>
+                <KeyboardAwareScrollView
+                  // allows opening links from search results without closing keyboard first
+                  keyboardShouldPersistTaps="handled"
+                  showsVerticalScrollIndicator={false}
+                  stickyHeaderIndices={[3]}
+                  refreshControl={
+                    <RefreshInvalidate invalidateKeys={invalidateKeys} />
+                  }
                 >
-                  <Text style={{ marginTop: 0, fontSize: 30, marginRight: 0 }}>
-                    OSU Golf Team
+                  <View style={{ alignItems: "center" }}>
+                    <Image
+                      uri={currentTeamData.pfp}
+                      style={{ marginTop: 0, width: 131, height: 75 }}
+                    />
```

Comment:
Do we want this to be a circle for the display? I attached a screenshot of an image I uploaded.
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/29dd5378-862d-4a2a-9cb4-62f43edbbabf)